docs(contribute): added to new docs (#1786)
latticexyz/mud: next-docs/pages/contribute.mdx

## Contribute

> We'd love your support in improving MUD! [The MUD monorepo](https://github.com/latticexyz/mud) includes all of MUD's source code, and pull requests are always welcome. To discuss new features or changes [join our Discord](https://lattice.xyz/discord).

Reviewer: ```suggestion
We'd love your support in improving MUD! [The MUD monorepo](https://github.com/latticexyz/mud) includes all of MUD's source code, and pull requests are always welcome.
To discuss new features or changes [join our Discord](https://lattice.xyz/discord).

You can find a list of good first issues (that don't require a lot of context on the full system) at [contribute.mud.dev](https://contribute.mud.dev).
```

## Contribute

> We'd love your support in improving MUD! This monorepo includes all of MUD's source code, and pull requests are always welcome. To discuss new features or changes [join our Discord](https://lattice.xyz/discord).

Reviewer: ```suggestion
We'd love your support in improving MUD! [The MUD monorepo](https://github.com/latticexyz/mud) includes all of MUD's source code, and pull requests are always welcome. To discuss new features or changes [join our Discord](https://lattice.xyz/discord).
```

### Local development setup

> The following steps are only necessary if you want to contribute to MUD. To use MUD in your project, [set up a new project with the MUD cli](/quick-start).

Reviewer: ```suggestion
The following steps are only necessary if you want to contribute to MUD. To use MUD in your project, [set up a new project with the MUD CLI](/quick-start).
```

1. Ensure you have these applications installed:

   | Software                                                                                    | Tested with version |
   | ------------------------------------------------------------------------------------------- | ------------------- |
   | [`curl`](https://curl.se/)                                                                  | 7.74.0              |
   | [`node`](https://nodejs.org/en/download/package-manager)                                    | 18.17.0             |
   | [`pnpm`](https://pnpm.io/)                                                                  | 8.6.11              |

>    | [`go` (at least version 1.20)](https://go.dev/dl/)                                          | 1.20.7              |
>    | [`make`](https://www.gnu.org/software/make/)                                                | 4.3                 |
>    | [`protoc`](https://grpc.io/docs/protoc-installation/#install-pre-compiled-binaries-any-os/) | 3.15.8              |

Reviewer: ```suggestion
```

these shouldn't be needed anymore since the `services` package is deprecated and not included in the build process anymore

1. Download the source code for MUD.

   ```sh copy
   git clone https://github.com/latticexyz/mud.git
   cd mud
   ```

1. Install and build packages.

   ```sh copy
   pnpm install
   pnpm build
   ```

### Pull requests

> MUD follows the [conventional commit specification](https://www.conventionalcommits.org/en/v1.0.0/) for commit messages and PR titles.

Reviewer: ```suggestion
MUD follows the [conventional commit specification](https://www.conventionalcommits.org/en/v1.0.0/) for PR titles.
```

> Please keep the scope of your PR small (rather open multiple small PRs than one huge PR) and follow [the conventional commit spec](https://www.conventionalcommits.org/en/v1.0.0/).

Reviewer: ```suggestion
Please keep the scope of your PR small (rather open multiple small PRs than one huge PR).
